runtime/doc/lsp.txt

```diff
@@ -1235,4 +1235,22 @@ trim_empty_lines({lines})                    *vim.lsp.util.trim_empty_lines()*
 try_trim_markdown_code_blocks({lines})
                 TODO: Documentation
 
+validate_lsp_position({pos})            *vim.lsp.util.validate_lsp_position()*
+                TODO: Documentation
+
+                                                   *vim.lsp.util.formatexpr()*
```

Comment:
why is it on `util`? we have `vim.lsp.omnifunc()` , and `vim.lsp.formatexpr` seems appropriate